package.json

```
@@ -2,10 +2,10 @@
   "name": "@openinf/openinf.github.io",
   "private": "true",
   "description": "The OpenINF website and other static resources",
-  "packageManager": "pnpm@8.14.3",
+  "packageManager": "pnpm@8.15.0",
```

```suggestion
  "packageManager": "pnpm@8.15.0+",
```

> The hash is optional but strongly recommended as a security practice.
> &mdash;https://github.com/nodejs/corepack#when-authoring-packages

we were planning to add the hash after the first automated @renovate-bot update when we initially implemented the field; the real reason why we want this is actually since the package.json schema validator complains w/o having it &mdash; not yet sure on its status